.github/ISSUE_TEMPLATE/end_game.md

```diff
@@ -69,10 +69,12 @@ assignees: ''
   - [ ] [Spark Data Connector](https://github.com/spiceai/quickstarts/blob/trunk/spark/README.md)
   - [ ] [Supabase Data Connector](https://github.com/spiceai/quickstarts/blob/trunk/supabase/README.md)
   - [ ] [Spice.ai Cloud Platform Data Connector](https://github.com/spiceai/quickstarts/blob/trunk/spiceai/README.md)
+  - [ ] Update the RC status of any connectors based on the [criteria](/docs/criteria/connectors/rc.md)
```

Comment:
this is a dead link until my PR merges 🙂

Reply:
Yep, I thinking ahead to the future :)